packages/next/src/client/components/react-dev-overlay/server/middleware-webpack.ts

```diff
@@ -471,9 +471,19 @@ async function getOriginalStackFrame({
     },
   })
 
+  let defaultNormalizedStackFrameLocation = frame.file
```

Comment:
I noticed this chunk of code seems to be re-used https://github.com/vercel/next.js/pull/76742/files#diff-79b354271f807079a015a2d9190c5c601f3f3e7eb2aeb390cc11b3e9fd84b006R350-R360

Should we create a re-usable fn for this?